src/index.sass

```diff
@@ -47,4 +50,50 @@ code
   display: flex
   align-items: center
   justify-content: center
-  width: 100%
\ No newline at end of file
+  width: 100%
+
+h1
+  font-family: 'Roboto'
+  font-style: normal
+  font-weight: 400
+  font-size: 24px
+  line-height: 28px
+
+#userHeading
+  position: absolute
+  width: 95px
+  height: 28px
+  left: 345px
+  top: 132px
+
+#userTable
+  position: absolute
+  left: 345px
+  top: 200px
```

Comment:
Oksettavaa. Käytäppä flexboxia 

https://css-tricks.com/snippets/css/a-guide-to-flexbox/